### PR TITLE
PLATSUP-1032 Add ZPS to platform docs

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/PLATSUP-1032

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1712171037840519) and give us a `:review-pr:` emoji.

File: docs/_publish/launch-process/zps.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: Zapier Partner Sandbox
+order: 4
```
Comment (marinahand) [inline docs/_publish/launch-process/zps.md:3]: if this is to be 4, could you update `user-help` to be 5 order?

File: docs/_publish/launch-process/zps.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- 2,000 free tasks per month
+    - Actions for Zapier apps do not count towards task usage
+- 5-minute update time for polling Zaps
+- Multi-step Zaps
+- 5 Premium Integrations
+- Paths (3 paths wide by 3 paths deep)
+
+Zapier Partner Sandbox does not include premium features for Zapier Tables or Zapier Interfaces.
+
+## Who’s eligible for Zapier Partner Sandbox?
```
Comment (marinahand) [inline docs/_publish/launch-process/zps.md:22]: Who is eligible (suggestion :) )

File: docs/_publish/launch-process/zps.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Any Admin or Collaborator on a public integration team can request access to Zapier Partner Sandbox.
```
Comment (marinahand) [inline docs/_publish/launch-process/zps.md:23]: to confirm, this is unlimited? so they can have multiple admins on this plan? if so, I'd suggest rewording to:

Each Admin or Collaborator on a public integration team can request access to Zapier Partner Sandbox.

**Review by marinahand — APPROVED** [review]
